docs/Modding/sound.rst

```diff
@@ -26,17 +30,16 @@ There are two kinds of sound tags:
 * defined in the rulesets
 * defined in the program code
 
-While the former can be chosen freely the latter can't be changed.
+While the former can be chosen freely the latter cannot be changed.
 
-The sound tags associated with improvements (wonders and normal buildings), unit movements and unit fights
-have to be set in the rulesets. Freeciv21 just hands these sound tags over to the client where they are
+The sound tags associated with improvements (wonders and normal buildings), unit movements, and unit combat
+have to be set in the rulesets. Freeciv21 just hands these sound tags over to the game where they are
 translated into the filenames of the sound files via the soundspec file. Every soundspec should have generic
-sound tags for wonders ("w_generic"), normal buildings ("b_generic"), unit movements ("m_generic") and unit
-fights ("f_generic").
+sound tags for wonders (``w_generic``), normal buildings (``b_generic``), unit movements (``m_generic``), and
+unit fights (``f_generic``).
```

Comment:
```suggestion
unit combat (``f_generic``).
```

Reply:
in this case we keep it "fights" as that is what the prefix refers to. otherwise i would agree with you.

Reply:
I suggested this because you changed "fights" to "combat" at the beginning of this paragraph. I suppose I should've explained this